conf/application.conf

```diff
@@ -132,10 +132,8 @@ auditing {
   }
 }
 
-ip-whitelist {
-  enabled: false
-  list: ["127.0.0.1"]
-  exclude: ["/ping/ping", "/healthcheck"]
+whitelist {
```

Comment:
Are we going to have some known eoris for local/dev/qa/staging config?

Reply:
I think he's done logic to allow everyone if the whitelist is empty

Reply:
👍 +1  Missed the "isEmpty" check